packages/marko-web-omeda-identity-x/index.js

```diff
@@ -15,6 +16,10 @@ module.exports = (app, {
   omedaGraphQLClientProp = '$omedaGraphQLClient',
   omedaRapidIdentifyProp = '$omedaRapidIdentify',
 
+  omedaPromoCodeCookieName = 'oly_promo_src',
```

Comment:
Is the `oly_promo_src` name used by Omeda for something, or was it "made up?" If it isn't an official Omeda param, I'd like to change it to not start with `oly_` otherwise it's hard to tell what is supported by Omeda and what isn't